skt/__init__.py

```diff
@@ -15,12 +15,27 @@
 from email.errors import HeaderParseError
 import email.header
 import email.parser
-import os
 import re
 
 import requests
 
 
+def join_with_slash(base, *suffix):
+    """
+    Join parts of URL or path by slashes
+    Args:
+        base:    Base URL or path.
+        *suffix: Array of suffixes
```

Comment:
Could we name this argument `suffix_tuple`, perhaps? So it's clear it's a sequence, and the iteration below would look clearer? Also, could you please document the slash-stripping behavior, and keeping the trailing slash in particular? This is the main value of this function and it's not documented.

Reply:
@veruu do you agree with his suggestion?

Reply:
@danghai yes, sounds good. But if you do the renaming, please also change the docstring to say `Tuple of suffixes` instead of array, for consistency reasons